src/index.html

```diff
@@ -9,21 +9,23 @@
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/stars.scss">
   </head>
   <body>
     <div class="card" data-qa="card">
-      <img
-        src="./images/imac.jpeg"
-        alt="iMac"
-        class="card__image"
-      >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
```

Comment:
Maybe add more info here so it could appear on the tooltip

Reply:
thank you, that really makes code look better :)